MerlinAU.sh

```diff
@@ -1879,7 +1884,8 @@ Would you like to use the ROG build? (y/n)${NOct}\n"
         -F "firmver=${dottedVersion}" \
         -F "file=@${firmware_file}" \
         --cookie /tmp/cookie.txt > /tmp/upload_response.txt 2>&1 &
-        sleep 60
+        sleep 180
+        /sbin/service reboot
```

Comment:
Adding a reboot for some routers after 3 minutes of waiting if it didn't reboot itself.

Reply:
I just had an idea to improve this crucial point of the process. Let me code it and run some testing before I put it in the script.